pandas/core/arrays/categorical.py

```diff
@@ -2642,13 +2643,11 @@ def recode_for_categories(
     return new_codes
 
 
-def factorize_from_iterable(values):
+def factorize_from_iterable(values) -> Tuple[np.ndarray, Index]:
     """
     Factorize an input `values` into `categories` and `codes`. Preserves
     categorical dtype in `categories`.
 
-    *This is an internal function*
-
```

Comment:
This is needless info: by not being exposed outside of `pd.core` makes this an internal function. No need to be explicit.